### PR TITLE
Closed the previous pull request due to a small error

### PR DESCRIPTION
**Pull request by preethamcoder**
This is the real pull request. I fixed the .gitignore and created a new PR. Editing this from my phone, so I apologize for any errors. This has a working version of database models and an app.py file.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<<<<<<< HEAD
```
Comment (TracyMichaels) [inline app.py:1]: looks like some github artifacts got left over, maybe from copy-pasting from last pr? These need to be cleaned up before we can merge as they'll introduce bugs.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import UserMixin
+from dotenv import find_dotenv, load_dotenv
+from werkzeug.security import generate_password_hash, check_password_hash
+
+load_dotenv(find_dotenv())
+
+app = Flask(__name__)
+<<<<<<< HEAD
```
Comment (TracyMichaels) [inline app.py:17]: cleanup, same as above

File: app.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+database = os.getenv("database")
+=======
```
Comment (TracyMichaels) [inline app.py:19]: cleanup, same as above

File: app.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+database = os.getenv('database')
+>>>>>>> 735ecd8cf22a28e984f5b73a85e85633d05a4560
```
Comment (TracyMichaels) [inline app.py:21]: cleanup, same as above

File: app.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+app.config["SQLALCHEMY_DATABASE_URI"] = database
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = b"Nothing in here"
+db = SQLAlchemy(app)
+
+
+class User(db.Model, UserMixin):
+<<<<<<< HEAD
```
Comment (TracyMichaels) [inline app.py:29]: cleanup

File: app.py [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+    def __repr__(self):
+        """Represents the video commented and the user id"""
+        return f"<{self.video_id}:{self.user_id}>"
+
+    def comment(self):
+        """Returns the comment entered"""
+        return self.review
+
+=======
```
Comment (TracyMichaels) [inline app.py:66]: cleanup, same as above

File: app.py [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+	video_title = db.Column(db.String(50), nullable=False)
+	user_id = db.Column(db.Integer, nullable=False)
+	review = db.Column(db.String(280), nullable=False)
+
+	def __repr__(self):
+        	return f"<{self.movie_id}:{self.user_id}>"
+	
+	def comment(self):
+        	return self.review
+>>>>>>> 735ecd8cf22a28e984f5b73a85e85633d05a4560
```
Comment (TracyMichaels) [inline app.py:91]: cleanup, same as above

File: app.py [hunk 8]
```diff
@@ -0,0 +1,93 @@
+<<<<<<< HEAD
+# pylint: disable=no-member
+"""This is the main driver for the app, it has the database models and the routes"""
+
+=======
+>>>>>>> 735ecd8cf22a28e984f5b73a85e85633d05a4560
```
Comment (TracyMichaels) [inline app.py:6]: cleanup, same as above

File: app.py [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+	password = db.Column(db.String(350), nullable=False)
+
+	def __repr__(self):
+	        return f"<{self.id}:{self.first_name}>"
+
+    	def hash_pwd(self, passw):
+                self.password = generate_password_hash(passw)
+
+    	def check_password(self, passw):
+        	return check_password_hash(self.password, passw)
```
Comment (TracyMichaels) [inline app.py:78]: should these be here, looks like the contents of the user table?

File: app.py [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+	id = db.Column(db.Integer, primary_key=True, nullable=False)
+	video_title = db.Column(db.String(50), nullable=False)
+	user_id = db.Column(db.Integer, nullable=False)
+	review = db.Column(db.String(280), nullable=False)
+
+	def __repr__(self):
+        	return f"<{self.movie_id}:{self.user_id}>"
+	
+	def comment(self):
+        	return self.review
```
Comment (TracyMichaels) [inline app.py:90]: why are there 2 review tables?

**Review by TracyMichaels — CHANGES_REQUESTED** [review]
There are some breaking changes in here, especially with what appears to be some artifacts from github

File: app.py (hunk 10 above)
Comment (preethamcoder) [reply app.py:90]: These were caused by GitHub. I feel like we should be creating new pull requests every time we have a change. If we commit without it, the new junk is getting added. When I first started this branch, we had no issues like this.

**Pull request merged**